src/api/app/presenters/obs_factory/staging_project_presenter.rb

```diff
@@ -147,19 +147,19 @@ def testing_percentage
     # the lower the number, the earlier it is in the list - acceptable A first
     def sort_key
       main = case overall_state
-      when :acceptable then
+             when :acceptable then
         0
```

Comment:
What happened here? :thinking:

Reply:
@DavidKang 

> What happened here? :thinking:

EnforcedStyle is 'case', the default. Do you prefer 'end' EnforcedStyle?

Reply:
@eduardoj , I'm fine with the cop, but the content of each `when` should be indented

Reply:
Contents each 'when' indented. Please re-review again.